packages/components/src/resizable-box/index.js

```diff
@@ -102,4 +111,4 @@ function ResizableBox( {
 	);
 }
 
-export default ResizableBox;
+export default forwardRef( ResizableBox );
```

Comment:
IMO, forwarding refs should be standard for components from @wordpress/components.
I'm happy to see more of these :)